### PR TITLE
data: raghu-007

### PR DESCRIPTION
**Pull request by raghu-007**
## Check List (Check all the applicable boxes) 

- [x] My code follows the code style of this project.
- [x] My change requires changes to the documentation.
- [x] I have updated the documentation accordingly.
- [x] All new and existing tests passed.
- [x] This PR does not contain plagiarized content.
- [x] The title of my pull request is a short description of the requested changes.

## Note to reviewers

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/4341"></a>

### CODE REVIEW

File: data/raghu-007.json [hunk 1]
```diff
@@ -0,0 +1,19 @@
+{
+  "name": "RAGHU S (test)",
+  "displayStatsPublic": true,
+  "type": "personal",
+  "bio": "Software Engineer and Tech Entrepreneur",
+  "avatar": "https://github.com/raghu-007.png",
```
Comment (swagataroy30) [inline data/raghu-007.json:6]:
```suggestion
```

**Review by swagataroy30 — COMMENTED** [review]
In the recent update, we don't need to add avatar: anymore, it will auto-pick your GitHub profile picture. So, please remove that.
Updated docs: https://linkfree.eddiehub.io/docs/quickstart
Thank You!

File: data/raghu-007.json [hunk 2]
```diff
@@ -0,0 +1,19 @@
...
+      {
+      "name": "Follow me on Twitter",
+      "url": "https://twitter.com/raghu7_",
+      "icon": "FaTwitter"
+    },
+    {
+      "name": "Follow me on Github",
+      "url": "https://github.com/raghu-007",
+      "icon": "FaGithub"
+       },
```
Comment (swagataroy30) [inline data/raghu-007.json:17]:
```suggestion
       }
```

**Review by swagataroy30 — COMMENTED** [review]
You need to remove the comma from line 17. Thank You!

**Comment by raghu-007** [comment]
> You need to remove the comma from line 17. Thank You!

done

**Comment by raghu-007** [comment]
> In the recent update, we don't need to add avatar: anymore, it will auto-pick your GitHub profile picture. So, please remove that. Updated docs: https://linkfree.eddiehub.io/docs/quickstart Thank You!

done

**Comment by swagataroy30** [comment]
You can also add tags, tags will help people discover your profile. You can refer to this docs https://linkfree.eddiehub.io/docs/how-to-guides/tags.

**Review by swagataroy30 — APPROVED** [review]
LGTM !

**Review by Ananya2001-an — APPROVED** [review]

**Review by SaraJaoude — APPROVED** [review]
looks good

**Comment by raghu-007** [comment]
> looks good
tq

**Pull request merged**